spec/features/customers/show_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe "As a visitor when I vist '/customers/:id'" do
+  before :each do
+    @king_soops = Supermarket.create!(name: "King Soopers", location: "123 Food St.")
+    @bagel = @king_soops.items.create!(name: "Everything Bagel", price: 2)
+    @egg = @king_soops.items.create!(name: "Egg (not microwaved)", price: 1)
+    @bacon = @king_soops.items.create!(name: "Thick-Cut Bacon", price: 4)
+    @cheese = @king_soops.items.create!(name: "Estate Gouda", price: 3)
+    @hot_sauce = @king_soops.items.create!(name: "Swamp Mustard", price: 6)
+
+    @reed = @bagel.customers.create!(name: "Reed")
+
+    @reed.items << @egg
+    @reed.items << @bacon
+    @reed.items << @cheese
+  end
+
+  it "I see the customer name and a list of its items and the item details" do
+    visit "customers/#{@reed.id}"
+
+    expect(page).to have_content(@reed.name)
+    expect(page).to have_content(@bagel.name)
```

Comment:
Remember that we can use `within` blocks to better test where things are showing up on our page.